Lecture6/src/test/java/PositiveGrid.java

```diff
@@ -0,0 +1,76 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.annotations.AfterSuite;
+import org.testng.annotations.BeforeSuite;
+import org.testng.annotations.Test;
+
+import static org.testng.Assert.assertEquals;
+
+public class PositiveGrid extends BaseTest {
+
+	private WebDriver driver;
+
+	@BeforeSuite(alwaysRun = true)
+	public void setup() {
+		driver = new ChromeDriver();
+	}
+
+	@AfterSuite(alwaysRun = true)
+	public void wrapUp() {
+		driver.quit();
+	}
+
+	@Test(description = "Launches the PositiveGrid site")
+	public void launchSite() {
+		driver.get("https://www.positivegrid.com/");
+		assertEquals(driver.getTitle(), "Positive Grid | Guitar technology software, amplifiers, pedals.");
+	}
+
+	@Test(description = "Navigates to the login page")
+	public void openSignInPage() {
+		WebElement element = driver.findElement(By.className("nav-item-icon"));
+		element.click();
+		element = driver.findElement(By.className("data-login-PG-btn"));
+		element.click();
+		assertEquals(driver.getTitle(), "Positive Grid");
+	}
+
+	@Test(description = "Enters valid login data", dependsOnMethods = "openSignInPage")
+	public void loginAsAdmin() {
+		WebElement element = driver.findElement(By.id("inputEmail"));
```

Comment:
please move all locators to separate class with constants

Reply:
done